src/modules/commander/commander.cpp

```diff
@@ -3691,9 +3697,9 @@ bool
 stabilization_required()
 {
 	return (status.is_rotary_wing ||		// is a rotary wing, or
-		status.vtol_fw_permanent_stab || 	// is a VTOL in fixed wing mode and stabilisation is on, or
-		(vtol_status.vtol_in_trans_mode && 	// is currently a VTOL transitioning AND
-		 !status.is_rotary_wing));	// is a fixed wing, ie: transitioning back to rotary wing mode
+		vtol_status.fw_permanent_stab || 	// is a VTOL in fixed wing mode and stabilisation is on, or
+		(status.in_transition_mode && 		// is currently a VTOL transitioning AND
+		 !status.is_rotary_wing));		// is a fixed wing, ie: transitioning back to rotary wing mode
 }
```

Comment:
Ok.

Reply:
If we delete the old style/advanced mode switch configuration the need for fw_permanent_stab goes away.
We could also stop having MC manual === stabilized, either blocking out manual entirely for MC, or having manual -> stabilized in the state machine.